services/validators/envValidator.go

```diff
@@ -124,7 +124,7 @@ func getWorkspaceFromDB(dbHandle *sql.DB) (string, error) {
 }
 
 func createDBConnection() (*sql.DB, error) {
-	psqlInfo := misc.GetConnectionString(config.Default)
+	psqlInfo := misc.GetConnectionString(config.Default, "env-validator")
```

Comment:
```suggestion
	psqlInfo := misc.GetConnectionString(config.Default, "")
```

Reply:
Didn't understand why should we not send component name here?

Reply:
This comment was mostly done in relation to [this](https://github.com/rudderlabs/rudder-server/pull/4225#discussion_r1422526443) one